docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function stop {
+	kill $(ps -C ts3server -o pid= | awk '{ print $1; }')
+	exit
+}
+
+trap stop INT
+trap stop TERM
+
+test -d /data/files || mkdir -p /data/files && chown teamspeak:teamspeak /data/files
```

Comment:
Yea removing the `-p`from from `mkdir` and adding creating `/data`inside `Dockerfile`